addons/mass_mailing/static/src/js/mailing_contact_view_kanban.js

```diff
@@ -12,9 +12,12 @@ var MailingContactController = KanbanController.extend({
     }),
 
     _onImport() {
-        this.do_action('mass_mailing.mailing_contact_import_action', {
-            additional_context: this.renderer.state.context,
-        });
+        const context = this.renderer.state.context;
+        const actionParams = { additional_context: context };
+        if ((!context.default_mailing_list_ids) && (context.active_model === 'mailing.list')) {
+            actionParams.additional_context = { ...context, default_mailing_list_ids: context.active_ids };
```

Comment:
Can't you simply do `actionParams.additional_context.default_mailing_list_ids = context.active_ids` ?

Reply:
Yes indeed. Changed and added a default context in case it is undefined (context = this.renderer.state.context || {};)